examples/stm32/xively_demo_f4xx_nucleo_wifi/Inc/stm32l4xx_hal_conf.h

```diff
@@ -0,0 +1,365 @@
+/**

```

Comment:
Why do you have:

a)examples/stm32/xively_demo_f4xx_nucleo_wifi/Inc/stm32l4xx_hal_conf.h
b)examples/stm32/xively_demo_f4xx_nucleo_wifi/Inc/stm32l0xx_hal_conf.h

Shouldn't there be only one hal config ?